lib/pure/cookies.nim

```diff
@@ -39,9 +44,10 @@ proc parseCookies*(s: string): StringTableRef =
 
 proc setCookie*(key, value: string, domain = "", path = "",
                 expires = "", noName = false,
-                secure = false, httpOnly = false): string =
+                secure = false, httpOnly = false, 
+                maxAge = none(int), sameSite = SameSite.Lax): string =
```

Comment:
I can make `sameSite` default to `SameSite.None` and add overload with since. But it looks ugly.